src/mlte/measurement/cpu/local_process_cpu_utilization.py

```diff
@@ -228,3 +228,5 @@ def _get_cpu_usage(pid: int) -> float:
         return -1.0
     except ValueError:
         return -1.0
+    except FileNotFoundError as e:
+        raise RuntimeError("External program needed to get CPU usage was not found: " + str(e))
```

Comment:
To be consistent with string formatting across the codebase, can we refactor this to use f-strings:

```python
raise RuntimeError("External program needed to get CPU usage was not found: {e}")
```

Reply:
No problem